### ISSUE TITLE
Glitches and Freezing on Oculus Quest

### ISSUE THREAD

**Issue by sebjf** [RenderHeads/UnityPlugin-AVProVideo#386]
**Describe the bug**
On the Oculus Quest, spatialised audio TBE_8_2 encoded videos will glitch and sometimes freeze. The videos play OK on a desktop, and in the Oculus Gallery app. The glitches are not deterministic, neither is the freezing.

In our minimal working example nothing appears out of the ordinary when filtering the log by Unity AVPro or Audio, though in our actual application the log does sometimes show:

2020-06-09 16:13:31.844 13932-14098/com.company.product I/OvrAudio: : Invalid parameter

**Your Setup (please complete the following information):**
 - Unity version: 2019.2.10
 - AVPro Video version: 1.9.17t
 - Operating system version: Android
 - Device model: Quest
 - Video specs (resolution, frame-rate, codec, file size): 5760x2880~29.97 H264

**To Reproduce**
1. Copy the video file to the SD card under Downloads
2. Build and Deploy the Minimal Working Example to the Quest
3. Glitches will occur at least once per two playthroughs, usually more often

This has been reproduced on multiple Quests.

**Logs**
None that stand out.

**Screenshots**
Capturing video does not show the artefacts.

**Videos**

### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE
Instead email the link to us unitysupport@renderheads.com

**Comment by AndrewRH** [comment]
Hi @sebjf 

Thanks for reporting your issue.

Before we continue - could you try with the latest version of AVPro Video (1.11.2) and see whether you still have this behavior?   The version you're using is a year old and we have many a number of improvements since then.

Also, are you using OES mode for video playback?

Also, have you tried to determine what could be causing the glitch, such as removing the audio track, lowering the resolution of the video, using a newer version of Unity (and thus the Oculus Tools)?

Thanks,

**Comment by sebjf** [comment]
Hi Andrew,

I have just tried with the latest (1.11.1t - the one released yesterday on the Asset Store) and the issue is still there.

I tried OES mode using Demo 13 with our video, and the glitches disappear.
I will update our main project to use OES mode and see if this resolves it there too.

We know that adding spatialised audio has caused the glitch, as the videos were working OK before, but not why. The only mode we have been able to make work is TBE_8_2, so can't try, e.g. different numbers of channels. The glitches are non-deterministic, so I don't think its corruption in the files themselves.

Kind regards,
